tools/python3/phloping.py

```
@@ -178,11 +185,44 @@ def print_variance_across(scope_timer_filepath=None):
+    if len(sys.argv) == 1:
+        print("usage: $function_name -h")
+        print(
+            "available functions:\n\t"
+            + "\n\t".join([k for k, v in globals().items() if k.startswith("print_")]),
+        )
+    elif len(sys.argv) > 1:
         fn = sys.argv[1]
         sys.argv = [sys.argv[0]] + sys.argv[2:]
-        globals()[fn]()
+        if fn not in globals():
+            raise ValueError("requested function does not exist")
```

Ruff (0.15.2)

[warning] Avoid specifying long messages outside the exception class

(TRY003)